app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/MonthViewAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package de.tum.`in`.tumcampusapp.component.tumui.calendar
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import de.tum.`in`.tumcampusapp.R
+import de.tum.`in`.tumcampusapp.component.tumui.calendar.model.CalendarItem
+
+class MonthViewAdapter(private var daysOfMonth: ArrayList<String>, private var eventMap: Map<String, List<CalendarItem>>)
+    : RecyclerView.Adapter<MonthViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MonthViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        val view = inflater.inflate(R.layout.calendar_month_view_cell, parent, false)
+        val layoutParams = view.layoutParams
+        layoutParams.height = (parent.height * 0.166666666).toInt()
```

Comment:
what is `0.166666666`?? please add a constant to name this (and if complex enogh a comment)

Reply:
Each cell is one sixth of the parents height
```suggestion
        layoutParams.height = (parent.height * 1/6)
```